packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -23,21 +23,28 @@ const propTypes = {
    * Sets the maximum possible fill-value.
    */
   max: PropTypes.number,
+  /**
+   * Boolean check for gradient.
+   */
+  gradient: PropTypes.bool,
```

Comment:
Typically we name boolean props with a `is` or `has` prefix to indicate there are two possible options. IMO we should update this to `hasGradient`.

Reply:
Thanks. Changed prop name to `hasGradient`